samples/ballerina-by-example/examples/http-circuit-breaker/http-circuit-breaker.client.sh

```diff
@@ -0,0 +1,140 @@
+# The first 2 requests complete without any issue.
+$ curl -v http://localhost:9090/cb
+*   Trying 127.0.0.1...
+* Connected to localhost (127.0.0.1) port 9090 (#0)
+> GET /cb HTTP/1.1
```

Comment:
Do we need this info in the example? IMO following response is well enough and make sense. Otherwise, the sample would be lengthier.  
``` 
HTTP/1.1 200 OK
Content-Length: 14
Content-Type: text/plain
 
Hello World!!!
```

Reply:
Ack. Will change and see

Reply:
I think it's better to keep it as it is since this is a part of the actual output and the user would expect to see the exact output of whatever command we have put there, as a sample run.